projects/schematics/src/migrations/3_0/methods-and-properties-deprecations/data/checkout-config.service.migration.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  CHECKOUT_CONFIG_SERVICE,
+  CHECKOUT_STEP_SERVICE,
+  GET_CHECKOUT_STEP,
+  GET_CHECKOUT_STEP_ROUTE,
+  GET_CURRENT_STEP_INDEX,
+  GET_FIRST_CHECKOUT_STEP_ROUTE,
+  GET_NEXT_CHECKOUT_STEP_URL,
+  GET_PREVIOUS_CHECKOUT_STEP_URL,
+  SPARTACUS_STOREFRONTLIB,
+  STEP,
+  TODO_SPARTACUS,
+} from '../../../../shared/constants';
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+// projects/storefrontlib/src/cms-components/checkout/services/checkout-config.service.ts
+export const CHECKOUT_CONFIG_SERVICE_MIGRATION: MethodPropertyDeprecation[] = [
+  {
+    class: CHECKOUT_CONFIG_SERVICE,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    deprecatedNode: STEP,
```

Comment:
I do not see this in `release/2.0.x`, therefore remove this object

Reply:
I see it in release/2.0.x and release/2.1.x 

![Screen Shot 2020-11-05 at 4 32 46 PM](https://user-images.githubusercontent.com/10605724/98298804-93c16000-1f84-11eb-873d-310a2810e5c6.png)

Reply:
I see then the variable/value is wrong because it's `steps`, but this would be included in the component data for properties instead of here

Reply:
Yes it should be steps but this is the right place to include it.